bigstore/bigstore.py

```diff
@@ -336,6 +341,21 @@ def pull():
             sys.stderr.write('ERROR\n')
 
 
+def fetch(repository):
+    """
+    Pull metadata from a remote repository and merge it with our own.
+
+    :param repository: either a git url or name of a remote
+    """
+    assert repository
```

Comment:
An error message is already provided by argparse, this was just defensive coding.  I'll remove the assert.
```
$ git bigstore fetch
usage: git-bigstore fetch [-h] repository
git-bigstore fetch: error: too few arguments
```